### PR TITLE
feat(locales): Add Nepali(ne) translations

### PR DESCRIPTION
**Pull request by TheLearneer**
## What does this PR do?

This PR adds the necessary translation of the project in the Nepali (ne) language.

## Test Plan

N/A

## Related PRs and Issues

Issue - #1705 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yeah I did go through it. Also went through [the add-translation guide](https://github.com/appwrite/appwrite/blob/master/docs/tutorials/add-translations.md)

### CODE REVIEW

**Comment by TheLearneer** [comment]
Sample image:

image

File: app/config/locale/translations/ne.json [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
+    "emails.magicSession.signature": "{{project}} समूह",
+    "emails.recovery.subject": "पासवर्ड रिसेट",
+    "emails.recovery.hello": "नमस्ते {{name}}",
+    "emails.recovery.body": "{{project}}को पासवर्ड रिसेट गर्नको लागी यो लिंकमा जानुहोस।",
+    "emails.recovery.footer": "यदि तपाइँले आफ्नो पासवर्ड रिसेट गर्न सोध्नु भएको छैन भने तपाइँले यो सन्देश लाई बेवास्ता गर्न सक्नुहुन्छ।",
+    "emails.recovery.thanks": "धन्यवाद",
+    "emails.recovery.signature": "{{project}} समूह",
+    "emails.invitation.subject": "%s समूहको लागि %s मा निमन्त्रणा",
+    "emails.invitation.hello": "नमस्ते",
+    "emails.invitation.body": "{{owner}}ले तपाइँलाई {{project}}मा {{team}}को सदस्य बन्न आमन्त्रित गर्नु भएको छ। तेसैले तपाइँलाई यो सन्देश पठाइएको हो।",
```
Comment (lohanidamodar) [inline app/config/locale/translations/ne.json:26]:
```suggestion
    "emails.invitation.body": "{{owner}}ले तपाइँलाई {{project}}मा {{team}}को सदस्य बन्न आमन्त्रित गर्न चाहनु भएको छ। त्येसैले तपाइँलाई यो सन्देश पठाइएको हो।",
```

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Few suggestions.

File: app/config/locale/translations/ne.json [hunk 2]
```diff
@@ -2,31 +2,31 @@
     "settings.inspire": "\"बुद्धिमान हुनु को कला के लाई बेवास्ता गर्न जान्ने को कला हो।\"",
```
[the remark this answers is not in the document]
Comment (TheLearneer) [reply app/config/locale/translations/ne.json:2]: Oops that was there by default and I clearly missed it out 😅

File: app/config/locale/translations/ne.json [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+    "emails.verification.signature": "{{project}} समूह",
+    "emails.magicSession.subject": "लगइन",
+    "emails.magicSession.hello": "नमस्ते,",
+    "emails.magicSession.body": "लगइन गर्नको लागी यो लिंकमा जानुहोस।",
+    "emails.magicSession.footer": "यदि तपाइँले यो इमेल प्रयोग गरेर लगइन गर्न सोध्नु भएको छैन भने तपाइँले यो सन्देश लाई बेवास्ता गर्न सक्नुहुन्छ।",
+    "emails.magicSession.thanks": "धन्यवाद",
+    "emails.magicSession.signature": "{{project}} समूह",
+    "emails.recovery.subject": "पासवर्ड रिसेट",
+    "emails.recovery.hello": "नमस्ते {{name}}",
+    "emails.recovery.body": "{{project}}को पासवर्ड रिसेट गर्नको लागी यो लिंकमा जानुहोस।",
```
Comment (christyjacob4) [inline app/config/locale/translations/ne.json:20]:
```suggestion
    "emails.recovery.body": "{{project}} को पासवर्ड रिसेट गर्नको लागी यो लिंकमा जानुहोस।",
```

Please keep the templates consistent with the English translations file and add the missing spaces in the other strings as well

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/ne.json (hunk 3 above)
Comment (lohanidamodar) [reply app/config/locale/translations/ne.json:20]: @christyjacob4 In the Nepali language, space doesn't make sense there.

**Pull request merged**